src/components/LobbyScreen.vue

```diff
@@ -94,7 +96,7 @@ export default {
 			if (diff > -45000 && diff < 45000) {
 				return t('spreed', 'The meeting will start soon')
 			}
-			return this.timerInMoment.fromNow()
+			return futureRelativeTime(this.timerInMoment)
```

Comment:
You should pass ms, not the Moment object
```suggestion
			return futureRelativeTime(this.timerInMoment.valueOf())
```

Yet it still doesn't show minutes after update (from 6 minutes to 5), can't get why: 
![image](https://github.com/nextcloud/spreed/assets/93392545/0cb0e313-73c2-438e-8d79-ab989342d389)